cmd/influx_inspect/help/help.go

```diff
@@ -0,0 +1,41 @@
+package help
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+// Command displays help for command-line sub-commands.
+type Command struct {
+	Stdout io.Writer
+}
+
+// NewCommand returns a new instance of Command.
+func NewCommand() *Command {
+	return &Command{
+		Stdout: os.Stdout,
+	}
+}
+
+// Run executes the command.
+func (cmd *Command) Run(args ...string) error {
+	fmt.Fprintln(cmd.Stdout, strings.TrimSpace(usage))
+	return nil
+}
+
+const usage = `
+Usage: influx_inspect [[command] [arguments]]
+
+The commands are:
+
+    dumptsm              dumps low-level details about tsm1 files.
+    export               exports a tsm file to line protocol
```

Comment:
This operates at a shard level now `data` and `wal` dirs, right?  It's not just single a tsm file anymore.